suites/pacific/cephadm/tier-2-mon-scenarios.yaml

```diff
@@ -0,0 +1,96 @@
+#===============================================================================================
+# Conf: conf/pacific/cephadm/tier-2_6node_5mons.yaml
+# Test cases for cluster service operation
+#    - Bootstrap the cluster with 5 Mons
+#    - Scale down the Mons from 5 to 3
+#    - Check the OSD config file if the Mons are updated there
+#
+#===============================================================================================
+tests:
+  - test:
+      abort-on-fail: true
+      desc: Install software pre-requisites for cluster deployment.
+      module: install_prereq.py
+      name: setup pre-requisites
+
+  - test:
+      abort-on-fail: true
+      config:
+        steps:
+          - config:
+              command: bootstrap
+              service: cephadm
+              args:
+                mon-ip: node1
+          - config:
+              command: add_hosts
+              service: host
+              args:
+                attach_ip_address: true
+                labels: apply-all-labels
+          - config:
+              command: apply
+              service: osd
+              args:
+                all-available-devices: true
+          - config:
+              command: apply
+              service: rgw
+              pos_args:
+                - rgw.1
+              args:
+                placement:
+                  label: rgw
+          - config:
+              args:
+                - "ceph fs volume create cephfs"
+              command: shell
+          - config:
+              args:
+                placement:
+                  label: mds
+              base_cmd_args:
+                verbose: true
+              command: apply
+              pos_args:
+                - cephfs
+              service: mds
+          - config:
+              args:
+                - "ceph osd pool create rbd"
+              command: shell
+          - config:
+              args:
+                - "rbd pool init rbd"
+              command: shell
+      desc: bootstrap and deploy services.
+      destroy-cluster: false
+      polarion-id: CEPH-83573713
+      module: test_cephadm.py
+      name: Deploy cluster using cephadm
+
+  - test:
+      name: Scale down Monitor with placement
+      desc: Scale down monitor nodes to 3 with placement
+      module: test_mon.py
+      abort-on-fail: true
+      polarion-id: CEPH-83573732
+      config:
+        command: apply
+        service: mon
+        base_cmd_args:
+          verbose: true
+        args:
+          placement:
+            nodes:
+              - node1
+              - node3
+              - node5
+            sep: ";"    # separator to be used for placements
+
+  - test:
```

Comment:
Can we add this TC to - https://github.com/red-hat-storage/cephci/blob/master/suites/pacific/cephadm/test-raw_osd-deployment.yaml ?

Reply:
HI Vaibhav, I have used the existing conf file having 5 mons as suggested. But with that config file,this suite will not work as it is creating OSD on node 1 and we don't have any OSD in Node 1 as per config